tests/tests/test-democracy/test-democracy-referendum.ts

```diff
@@ -100,4 +100,43 @@ describeDevMoonbeam("Democracy - Referendum", (context) => {
     let parachainBondInfo = await context.polkadotApi.query.parachainStaking.parachainBondInfo();
     expect(parachainBondInfo.account.toString()).to.equal(ZERO_ADDRESS);
   });
+
+  it("should be votable while staked", async function () {
+
+    // transfer funds into a new account so we know there will be no locks/reserves/etc
+    const randomAccount = generateKeyingPair();
+
+    await context.createBlock(
+      context.polkadotApi.tx.balances.transfer(randomAccount.address, 100_000_000_000_000_000_000n)
```

Comment:
```typescript
100n * GLMR
``` 
is a bit easier to read here

Reply:
I think Alan took care of this when he improved the tests :+1: